includes/functions/core.php

```diff
@@ -72,3 +72,42 @@ function activate() {
 function deactivate() {
 
 }
+
+/**
+ * Enqueue scripts for front-end.
+ *
+ * @uses wp_enqueue_script() to load front end scripts.
+ *
+ * @since 0.1.0
+ *
+ * @return void
+ */
+function scripts() {
+
+	wp_enqueue_script(
+		'frontend',
+		TENUP_SCAFFOLD_TEMPLATE_URL . "/dist/js/frontend.min.js",
+		[],
+		false,
```

Comment:
@timwright12 We should set some kind of version number for the enqueues, even if we just use `TENUP_SCAFFOLD_VERSION`. It'll help with cache busting when we deploy upgrades.